test/base.py

```diff
@@ -33,21 +35,67 @@
 from .extended_equality import is_equivalent
 
 # Fail tests that take longer than this
-TEST_TIMEOUT = os.environ.get("TEST_TIMEOUT", 60)
+TEST_TIMEOUT = int(os.environ.get("TEST_TIMEOUT", 60))
+# Use testtools by default as a (mostly) drop in replacement for
+# unittest's TestCase. This will enable the fixtures used for capturing stdout
+# stderr, and pylogging to attach the output to stestr's result stream.
+USE_TESTTOOLS = os.environ.get("QE_USE_TESTOOLS", "TRUE") == "TRUE"
```

Comment:
I made testtools a required import. It is a dependency of `fixtures` any way. Still, I personally had a niche use case for the tests not deriving from `testtools.TestCase` so I left that a possibility. If it becomes a problem we can remove it. My niche case is that for some things I like to use pytest plugins for debugging. pytest can run `unittest` tests but it can not run `testtools` tests.